app/javascript/pages/DirectoryPage.tsx

```diff
@@ -11,13 +12,20 @@ interface DirectoryProps {
   assetPaths: unknown
 }
 
-const DirectoryPage = (props: DirectoryProps) => (
-  <Layout>
-    <Head>
-      <title>{t("t.dahliaSanFranciscoHousingPortal")}</title>
-    </Head>
-    Is Rental: {String(props.isRental)}
-  </Layout>
-)
+const DirectoryPage = (props: DirectoryProps) => {
+  useEffect(() => {
+    // todo: remove this. it's just for display purposes
+    void getRentalListings().then((listings) => console.log(listings))
```

Comment:
@akegan should we keep it for not to make developing frontend a bit easier?

Reply:
yes let's keep it for now since we don't have any other way of testing this PR and seeing the data while we develop the rest of the page.